### ISSUE TITLE
NSD support

### ISSUE THREAD

**Issue by ManDude** [cbhacks/CrashEdit#23]
Opening an NSD should make CrashEdit show its content similar to the way it shows NSFs: Instead of chunks you got chunk index, spawn point index, etc. and inside those are the EIDs and CIDs (and spawn point garbage in the spawn point section, this is made in a weird way though)

**Comment by ughman** [comment]
Yeah, that's a good idea. And then instead of Patch NSD there could be something like "Rebuild Index" or something.
